jsf/subsystem/src/main/java/org/jboss/as/jsf/logging/JSFLogger.java

```diff
@@ -104,8 +104,8 @@
     DeploymentUnitProcessingException invalidDefaultJSFImpl(String defaultJsfVersion);
 
     @LogMessage(level = ERROR)
-    @Message(id = 15, value = "Failed to parse %s, phase listeners defined in this file will not be available")
-    void phaseListenersConfigParseFailed(VirtualFile facesConfig);
+    @Message(id = 15, value = "Failed to parse %s, JSF artifacts defined in this file will not be available")
+    void jsfArtifactsConfigParseFailed(VirtualFile facesConfig);
```

Comment:
Does this new message describe the same situation as the old message, just with slightly different wording?  If not a new message should probably be used as the message ids are meant to be dictionary keys with the entry they point to (say a kbase, our own or a users) not radically changing.

I recognize there can be some gray area here; if there is whatever @fjuma thinks best is fine with me.

Reply:
Previously there was one message for managed-beans parsing error and another one for phase-listeners (two different parsing executions). Now there is one parsing method for all the artifacts, so it's very similar but not exactly the same.

I just re-used one of the previous messages and commented out the other one. I really didn't think too much about this, I just re-used it. If you prefer a new ID/message and commenting out the previous two just let me know.

Thanks!

Reply:
Thanks. The parsing error message change is fine with me.

This next question is somewhat OT to this PR as it's about the existing code, but this message doesn't really tell the user anything about the problem. Should the actual error that leads to it be logged at DEBUG or something?

Reply:
IMHO the message is OK at error level, or at least at warning. More or less that message means the faces-config file is incorrect and it is not going to be fully parsed for managed-beans or other artifacts. Why do you say it doesn't say anything about the error? Would it be better to log the stack-trace too?

Reply:
For the moment and after reading your comments I have used the first message instead of the second (the first one is the important one, so better re-use the first one and comment out the second). It's just a slightly different wording but same ID and error level. I have rebased to current master too.

Reply:
@rmartinc Just to close out this discussion, which I failed to come back to. Yes, I was referring to logging a stack trace at DEBUG.

That's not urgent; if this passes CI agains it's ready for merge.